lib/core/services/services_locator.dart

```diff
@@ -22,6 +22,11 @@ import 'package:snap_jobs/authentication_and_login_features/presentation/control
 import 'package:snap_jobs/core/network/api_constants.dart';
 import 'package:snap_jobs/core/network/base_http_client.dart';
 import 'package:snap_jobs/core/network/network_info.dart';
+import 'package:snap_jobs/profile_feature/data/data%20source/cashedatasource.dart';
+import 'package:snap_jobs/profile_feature/data/data%20source/networkdatasource.dart';
```

Comment:
the %20 in "data source" in address happens when you have a space in file or folder name.
please change it and remove it as it makes errors